index.html

```diff
@@ -51,6 +51,123 @@
                 </ul>
             </div>
         </section>
+        <section id="works" class="works">
+             <div class="work">
+                <img alt="work one" class="work-image" src="./assets/images/works.svg"/>
+                <h2 class="work-title">Tonic</h2>
```

Comment:
- [ ] Kindly use a header tag hx, This way will be semantically correct and the screen readers.